watertap/costing/watertap_costing_package.py

```diff
@@ -857,6 +858,50 @@ def cost_crystallizer_by_volume(blk):
             )
         )
 
+    @staticmethod
+    def cost_ion_exchange(blk):
+        """
+        Volume-based capital cost for Ion Exchange
+        """
+        resin_dict = {
+            "strong_acid_gel_1": 3689,
+            "strong_acid_macro": 6255,
+            "strong_base_styrenic_gel_1": 5214,
+            "strong_base_styrenic_gel_2": 6116,
+            "strong_base_styrenic_macro_1": 7298,
+            "strong_base_styrenic_macro_2": 7810,
+            "strong_base_polyacrylic": 8658,
+            "strong_base_nitrate": 6116,
+        }  # cost of resin per m3, adapted to $/m3 from EPA models
+        make_capital_cost_var(blk)
+        num_cols = 2
+        col_cost_params = [9120, 0.49]
+        resin_cost_params = resin_dict["strong_acid_macro"]

```

Comment:
```suggestion
        resin_cost_params = resin_dict[resin_cost_type]

```

Reply:
@kurbansitterley like @TimBartholomew mentioned during our call, you could probably follow the example of the costing the RO membrane, where we have costs for whether a high-pressure module is being used as opposed to "conventional" RO module.